### DISCUSSION TITLE
Do we have a support to create Kubeflow experiment creation using k8s manifest?

### DISCUSSION THREAD

**Discussion by harsha1163** [deployKF/deployKF#214] [Questions & Answers]
Can we create kubeflow pipeline experiment using kubernetes manifest? do we have any custom resource to do it? 

<img width="249" alt="image" src="https://github.com/user-attachments/assets/d9117ec9-5ede-442b-a523-8b71192a0ffe">

**Comment by thesuperzapper** [comment]
@harsha1163 KFP does not currently have a CRD for experiments. I guess it would be similar to the request for a CRD that manages recurring runs (see: https://github.com/kubeflow/pipelines/issues/6001).

However, deployKF has a solution to this if you are willing to set up a git-ops process, see the reference repo in:

- https://github.com/deployKF/kubeflow-pipelines-gitops

__NOTE:__ right now that example has not been updated for KFP V2, so if you end up using it, please raise a PR to help update it for KFP v2.
